App/SB/views/PhotoDetail/index.js

```diff
@@ -45,7 +42,9 @@ class PhotoDetail extends Component {
             <Image style={styles.toolbarAddIcon} source={require('./statics/icon-add.png')}/>
           </TouchableOpacity>
           <Image style={styles.toolbarDownloadIcon} source={require('./statics/icon-download.png')}/>
-          <Image style={styles.toolbarShareIcon} source={require('./statics/icon-share.png')}/>
+          <TouchableOpacity onPress={params.getPublicLink}>
```

Comment:
There's a nice library we were using called react-native-header-buttons that makes this a lot nicer, you wouldn't have to be "creating" "buttons" by using `Image` and `TouchableOpacity`. Not sure if you want to change now, could do it later.

Reply:
ah nice. will start using